charts.d/mysql.chart.sh

```diff
@@ -10,136 +10,26 @@
 mysql_update_every=5
 mysql_priority=60000
 
-declare -A mysql_cmds=() mysql_opts=() mysql_ids=()
+declare -A mysql_cmds=() mysql_opts=() mysql_ids=() mysql_data=()
 
-mysql_exec() {
-	local ret
-
-	"${@}" -s -e "show global status;"
-	ret=$?
+mysql_get() {
+	local arr
+	local oIFS="${IFS}"
+	mysql_data=()
+	IFS=$'\t'$'\n'
+	arr=($("${@}" -e "SHOW GLOBAL STATUS WHERE value REGEXP '^[0-9]';" | egrep "^(Bytes|Slow_|Queri|Handl|Table|Selec|Sort_|Creat|Conne|Abort|Binlo|Threa|Innod|Qcach|Key_|Open)" ))
```

Comment:
It runs with and without `-s` and as I recall there was minor time penalty when running mysql with it.